web/gui/README.md

```diff
@@ -1,108 +1,106 @@
 
-```sh
-cd web/gui
-make
-```
+By default, Netdata starts a web server for its dashboard at port `19999`. Open up your web browser of choice and navigate to `http://SERVER-IP:19999`, or `http://localhost:19999` on `localhost`.
+
+Netdata uses an [internal, static-threaded web server](../server/) to host the HTML, CSS, and JavaScript files that make up the standard dashboard. You don't have to configure anything to access it, although you can adjust [your settings](../server/#other-netdataconf-web-section-options) in the `netdata.conf` file, or run Netdata behind an Nginx proxy, and so on.
+
+## Navigating the standard dashboard
+
+Beyond charts, the standard dashboard can be broken down into three key areas:
+
+1. [**Sections**](#sections)
+2. [**Menus/submenus**](#menus)
+3. [**Nodes menu**](#nodes-menu)
+
+![Annotated screenshot of the dashboard](https://user-images.githubusercontent.com/1153921/62810777-ef681980-bab3-11e9-9310-ec7a0c9f4faa.png)
+
+### Sections
+
+Netdata is broken up into multiple **sections**, such as **System Overview**, **CPU**, **Disk**, and more. Inside each section you'll find a number of charts, broken down into [contexts](../README.md#contexts) and [families](../README.md#families).
+
+An example of the **Memory** section on a Linux desktop system.
+
+![Screenshot of the Memory section of the Netdata dashboard](https://user-images.githubusercontent.com/1153921/62811044-1410c100-bab5-11e9-91cc-414cf6c23985.png)
+
+All sections and their associated charts appear on a single "page," so all you need to do to view different sections is scroll up and down the page. But it's usually quicker to use the [menus](#menus).
+
+### Menus
 
-After every change in the `src` directory, the `dashboard.js` file should be regenerated and commited to the repository.
+**Menus** appears on the right-hand side of the standard dashboard. Netdata generates a menu for each section, and menus link to the section they're associated with.
 
-## Custom Dashboards
+![Screenshot of the menu](https://user-images.githubusercontent.com/1153921/62811361-38b96880-bab6-11e9-8d41-4d9b29778e86.png)
 
-For information on creating custom dashboards, see **[Custom Dashboards](custom/)** and **[Atlassian Confluence Dashboards](confluence/)**
+Most menu items will contain several **submenu** entries, which represent any [families](../README.md#families) from that section. Netdata automatically generates these submenu entries.
 
-## Supported chart libraries
+Here's a **Disks** menu with several submenu entries for each disk drive and partition Netdata recognizes.
 
-- Dygraph
-- jQuery Sparkline
-- Peity
-- Google Charts
-- Morris
-- EasyPieChart
-- Gauge.js
-- D3
-- C3
+![Screenshot of some submenus](https://user-images.githubusercontent.com/1153921/62811428-70281500-bab6-11e9-9a61-ecd0a7aba453.png)
 
-### Dygraph
+### Nodes menu
```

Comment:
The nodes menu will also show the master netdata node and all slave nodes streaming to that master, if you have configured streaming. ../../streaming
![Screenshot from 2019-08-13 09-52-39](https://user-images.githubusercontent.com/43294513/62924643-d2cc2a00-bdb0-11e9-8ec7-dac79a21a97d.png)